liquibase-core/src/main/java/liquibase/sql/visitor/AbstractSqlVisitor.java

```diff
@@ -76,7 +76,12 @@ public void setLabels(Labels labels) {
 
     @Override
     public CheckSum generateCheckSum() {
-        return CheckSum.compute(new StringChangeLogSerializer().serialize(this, false));
+        String stringToMd5 = new StringChangeLogSerializer().serialize(this, false);
```

Comment:
Should `labels` and `labelsFilter` also be included in this change? Or are labels not included in the checksum calculation?

CC  @filipelautert